postgres-appliance/Dockerfile.build

```diff
@@ -390,6 +387,7 @@ RUN export DEBIAN_FRONTEND=noninteractive \
         && echo 4.0.0.dev0 > /usr/local/lib/python3.5/dist-packages/supervisor/version.txt; \
     fi \
     && pip3 install "patroni[kubernetes$EXTRAS]==$PATRONIVERSION" \
+    && pip3 install awscli \
```

Comment:
It adds another 45MB to the /usr directory and the total size of the image will become about 7MB bigger...
Why not just write one python script which will do everything? http://boto.cloudhackers.com/en/latest/s3_tut.html#storing-large-data